widgetssdk/src/main/java/com/glia/widgets/core/queue/GliaQueueRepository.java

```diff
@@ -150,9 +150,9 @@ public Completable observeQueueTicketStateChangeToUnstaffed() {
     @NonNull
     public RequestCallback<QueueTicket> emitOnQueueTicketChangeToUnstaffed(CompletableEmitter emitter) {
         return (queueTicket, e) -> {
-            if (queueTicket.getState() == QueueTicket.State.UNSTAFFED) {
-                emitter.onComplete();
-            }
+//            if (queueTicket.getState() == QueueTicket.State.UNSTAFFED) {
+//                emitter.onComplete();
+//            }
```

Comment:
Could you please explain the reason for disabling this?

Reply:
https://github.com/salemove/android-sdk-widgets/commit/ccca28d0c62f73021106e9e0bd8a900495ccfcb3